### ISSUE TITLE
selectedSegmentIndex is not changing its index

### ISSUE THREAD

**Issue by manomuthu-optisol** [alokc83/MASegmentedControl#8]
While I have tried to change my segment index programatically its not reflecting.

**Comment by basicemotions** [comment]
I have same problem, how to change index programmatically?

**Comment by ATrotsenko** [comment]
I have same problem, how to change index programmatically?

**Comment by alokc83** [comment]
Thanka for reporting. Let me look it up, what's happening

**Comment by alokc83** [comment]
@manomuthu-optisol  & @ATrotsenko  
Thank you so much for letting me know about this issue. 
I have fixed it. Until I update pod for v0.0.6, you can still use it as below: 
```ruby
pod 'MASegmentedControl', :git => 'https://github.com/alokc83/MASegmentedControl.git', :branch => 'v0.0.6'
```

**Comment by shoaibsadanaut** [comment]
@alokc83 When is ths v0.0.6 is coming. I need to add it to my podspec file. I can't use :git in podspec so kinda waiting for this. Thanks

**Comment by alokc83** [comment]
Fixed in 0.0.6
